src/app/common/default.controller.js

```diff
@@ -0,0 +1,52 @@
+
+(function() {
+  'use strict';
+  angular
+    .module('healthGuide')
+    .controller('defaultController', defaultController);
+
+  function defaultController(
+                      localForageService,
+                      sectionsService,
+                      sidenavService,
+                      userService
+                      ) {
+      "ngInject";
+      /*jshint validthis: true*/
+      var self = this;
+      self.state = 'default';
+
+      self.toggleSideNav = sidenavService.toggleList;
+      self.openLeftMenu = sidenavService.openLeftMenu;
+      self.closeSidenav = sidenavService.closeSidenav;
+
+      self.user = userService.user;
+      self.data = sectionsService.data[self.state];
+      self.calculate = calculate;
+
+      activate();
+
+      function activate() {
+        self.closeSidenav();
+        if (_.isEqual(userService.user, userService.mockUser)) {
```

Comment:
I can remove it only with ` if (_.isEqual(userService.user, userService.mockUser)).`


The reason is $http request [runs at the runblock ](https://github.com/VasylPosmit/health/blob/add-offline-mode/src/app/config.js#L42)asynchronously and return promise with userData.

I need to apply userData in controller (self.user = userService.user) **after** I promise resolved. I count that $http request took 1300 ms so made timeout for this time